activemodel/lib/active_model/attribute_methods.rb

```diff
@@ -370,6 +370,8 @@ def local_attribute_aliases # :nodoc:
       end
 
       private
+        attr_writer :local_attribute_aliases # :nodoc:
```

Comment:
Added to resolve these failures - https://buildkite.com/rails/rails/builds/98910#018a198e-03ae-4d10-820e-67f72d4b14c0. Also matches the signature for `attribute_aliases`, which might be better anyway.